packages/components/stamp/src/stamp.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import * as icons from '@commercetools-uikit/icons';
+import SpacingsStack from '../../spacings/spacings-stack';
+import SpacingsInline from '../../spacings/spacings-inline';
+import Stamp, { availableTones } from './stamp';
+
+const iconNames = Object.keys(icons);
+const numberOfIcons = iconNames.length;
+const getRandomIndex = (min: number, max: number) =>
+  Math.floor(Math.random() * (max - min + 1)) + min;
+
+const meta = {
+  title: 'Components/Stamp',
+  component: Stamp,
+  tags: ['autodocs'],
+} satisfies Meta<typeof Stamp>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    isCondensed: false,
+  },
+  parameters: { controls: { exclude: ['tone', 'children', 'icon', 'label'] } },
```

Comment:
If we exclude these props they won't be displayed in the props table so users won't know about them, right?

Reply:
Well, that's true, but keeping in mind how this story works, exposing those controls and allowing users to tweak them might create the impression that something is broken, since changing the values won't have any effect. We can leave them out, but then the story must be changed

Reply:
I think we should include this topic in our upcoming meeting.

Reply:
As discussed during our sync I used [this pattern](https://storybook.js.org/docs/essentials/controls#disable-controls-for-specific-properties) in the new `hideControls` helper. Now the controls are visible in the table but the value is not passed to the story